module/metrics/herocache.go

```diff
@@ -29,6 +29,10 @@ func NetworkReceiveCacheMetricsFactory(registrar prometheus.Registerer) *HeroCac
 	return NewHeroCacheCollector(namespaceNetwork, ResourceNetworkingReceiveCache, registrar)
 }
 
+func PublicNetworkReceiveCacheMetricsFactory(registrar prometheus.Registerer) *HeroCacheCollector {
+	return NewHeroCacheCollector("public_"+namespaceNetwork, ResourceNetworkingReceiveCache, registrar)
```

Comment:
```suggestion
	return NewHeroCacheCollector(namespacePublicNetwork, ResourcePublicNetworkingReceiveCache, registrar)
```

Should it be `ResourceNetworkingReceiveCache` or `ResourcePublicNetworkingReceiveCache`?